src/renderer/dx/DxFontInfo.h

```diff
@@ -74,4 +76,23 @@ namespace Microsoft::Console::Render
         // Indicates whether we couldn't match the user request and had to choose from a hardcoded default list.
         bool _didFallback;
     };
+
+    struct DxFontInfoHash
+    {
+        size_t operator()(const DxFontInfo& fontInfo) const noexcept
+        {
+            size_t h1 = std::hash<std::wstring_view>{}(fontInfo.GetFamilyName());
+            size_t h2 = std::hash<DWRITE_FONT_WEIGHT>{}(fontInfo.GetWeight());
+            size_t h3 = std::hash<DWRITE_FONT_STYLE>{}(fontInfo.GetStyle());
+            size_t h4 = std::hash<DWRITE_FONT_STRETCH>{}(fontInfo.GetStretch());
+            size_t h5 = std::hash<bool>{}(fontInfo.GetFallback());
+
+            auto combine = [](size_t seed, size_t v) {

```

Comment:
This is from `boost::hash_combine`. I wonder if I could just bring in the original boost code here, alongside the other boost code